### ISSUE TITLE
Grouping commands

### ISSUE THREAD

**Issue by krystof-k** [yannbertrand/macos-defaults#221]
Some settings change multiple keys. For example default Finder new window setting:

- `com.apple.finder.NewWindowTarget` = `PfAF` &  
  `com.apple.finder.NewWindowTargetPath` = `file:///System/Library/CoreServices/Finder.app/Contents/Resources/MyLibraries/myDocuments.cannedSearch`  
  for _Recents_
- `com.apple.finder.NewWindowTarget` = `PfHm` &  
  `com.apple.finder.NewWindowTargetPath` = `file:///Users/username/`  
  for home (`~/.`)
- `com.apple.finder.NewWindowTarget` = `PfDe` &  
  `com.apple.finder.NewWindowTargetPath` = `file:///Users/username/Desktop/`  
  for desktop (`~/Desktop/`)
- `com.apple.finder.NewWindowTarget` = `PfDo` &  
  `com.apple.finder.NewWindowTargetPath` = `file:///Users/username/Documents/`  
  for documents (`~/Documents/`)
- `com.apple.finder.NewWindowTarget` = `PfID` &  
  `com.apple.finder.NewWindowTargetPath` = `file:///Users/username/Library/Mobile%20Documents/com~apple~CloudDocs/`  
  for iCloud (`~/Library/Mobile%20Documents/com~apple~CloudDocs/`)
- `com.apple.finder.NewWindowTarget` = `PfVo` &  
  `com.apple.finder.NewWindowTargetPath` = `file:///`  
  for a (root) volume
- `com.apple.finder.NewWindowTarget` = `PfCm` &  
  `com.apple.finder.NewWindowTargetPath` deleted  
  for computer
- `com.apple.finder.NewWindowTarget` = `PfLo` &  
  `com.apple.finder.NewWindowTargetPath` = `file:///custom/path`  
  for a custom location

I believe it would make more sense to group the commands under some higher entity which would describe the overall behavior (or setting in the UI). That would be also future-proof when the underlying commands may change in future macOS versions, but the behavior may stay the same.

**Comment by yannbertrand** [comment]
Good point (see #206). I believe the `defaults.yml` file has made its time, it's limitating on several points.

I will simplify the whole project by just creating a markdown file for each page, no more `yml` to `md` compiler. I'll let you updated in this issue.
